justfile

```diff
@@ -77,7 +77,7 @@ _lint-toml:
 
 [no-exit-message]
 _lint-md:
-  markdownlint-cli2 "**/*.md" "#target" "#.github" "#**/*/astria-bridge-contracts/lib/**/*.md"
+  markdownlint-cli2 "**/*.md" "#target" "#.github" "#**/*/astria-bridge-contracts/lib/**/*.md" "#**/CHANGELOG.md"
```

Comment:
This looks like it's omitting changlogs from lint? We probably don't want to do that?